README.md

```diff
@@ -76,7 +76,7 @@ AEAD primitive in Java:
     Aead aead = AeadFactory.getPrimitive(keysetHandle);
 
     // 3. Use the primitive.
-    byte[] ciphertext = aead.encrypt(plaintext, aad);
+    byte[] ciphertext = aead.encrypt(plaintext, new byte[0] /* additionalData */);
```

Comment:
How about changing the comment to "associated data"?

That way, it matches the acronym (i.e. the AD from AEAD). Also, there's no need to use camel-case syntax within a comment.

Reply:
Or simply change the variable name (note that in the context of this example, the variable "plaintext" isn't initialized either).

```
byte[] ciphertext = aead.encrypt(plaintext, associatedData);
```

Reply:
What's about initializing both variables?

    String plaintext = "secretplaintext";
    byte associatedData = new byte[0];

Also there should be an explanation. Tink is for people who dont know crypto in detail so in README it says 'using associated_data as additional AEAD-input'. I think it is not described exactly enough to understand why you need associated_data, what it does and how to use it.

Reply:
Also in PRIMITIVE.md the explanation should be more specific. English is not my native language, so it's better if someone else does that.

Reply:
For now, the low hanging fruit is updating README.md to make the purposely terse usage example a bit clearer. I'll go ahead and merge this CL as is to accomplish this.

Moving forward, we're preparing to launch a site for Tink separate from the documentation embedded within the repository. Such a site will provide a logical place to expand on introductory content.

Thanks for the PR and the feedback.